src/tests/performance/connections.rs

```diff
@@ -199,15 +218,15 @@ async fn p002_connections_load() {
             let (exit_tx, exit_rx) = tokio::sync::oneshot::channel::<()>();
             synth_exits.push(exit_tx);
 
+            let sock = synth_sockets.remove(0);

```

Comment:
I just figured,
you could use this as an alternative approach to this for loop:

```rust
        // start synthetic nodes
        for socket in synth_sockets.into_iter() {
            let (exit_tx, exit_rx) = tokio::sync::oneshot::channel::<()>();
            synth_exits.push(exit_tx);

            let synth_handshaken = handshake_tx.clone();
            // Synthetic node runs until it completes or is instructed to exit
            synth_handles.push(tokio::spawn(async move {
                tokio::select! {
                    _ = exit_rx => {},
                    _ = simulate_peer(node_addr, synth_handshaken, socket) => {},
                };
            }));
```
to avoid having that unused `underscore` in `for _ in 0..synth_count` 😎
and avoid doing manually this part: `synth_sockets.remove(0);`

Reply:
True, changed this way :)